WeScan/Scan/FocusRectangle.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  FocusRectangle.swift
+//  WeScan
+//
+//  Created by Julian Schiavo on 16/11/2018.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import UIKit
+
+final class FocusRectangle: UIView {
```

Comment:
Can we add a line of text to explain that this is for the tap to focus UI like you have in this PR

Reply:
Resolved: 9d1e1e4